lib/src/controls/flyouts/content_manager.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:fluent_ui/fluent_ui.dart';
+
+class ContentManager extends StatefulWidget {
+  const ContentManager({
+    Key? key,
+    required this.content,
+  }) : super(key: key);
+
+  final WidgetBuilder content;
+
+  @override
+  State<ContentManager> createState() => _ContentManagerState();
+}
+
+class _ContentManagerState extends State<ContentManager> {
+  final GlobalKey key = GlobalKey();
+  Size size = Size.zero;
+
+  @override
+  void initState() {
+    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
```

Comment:
note: addPostFrameCallback makes the framework "lose" a frame as it needs to rebuild two times and as such it's not recommended. Is there really no other way to get the size from context without this? Maybe from within a callback or such

Reply:
Never found any other way to do this